pim-tool-front/src/app/confirmation-dialog/confirmation-dialog.component.scss

```diff
@@ -0,0 +1,14 @@
+.dialog-buttons {
+  display: flex;
+  justify-content: flex-end;
+  gap: 16px; /* Adjust the desired spacing between buttons */
+}
+
+.delete-button {
+  /* Your existing styles for the button */
+
+  /* Add the following styles to emphasize the button */
+  font-weight: bold;
+  color: #D2492A;
```

Comment:
[N] Create css variables and reuse the variable
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties